### PR TITLE
Update Consumer browser section

### PR DESCRIPTION
**Pull request by AJ-72**
Update the table for Firefox and Apple.

### CODE REVIEW

File: start.html [hunk 1]
```diff
@@ -41,10 +41,10 @@
  <tr><th>Browser</th> <th>Maintainer</th> <th>Versions Supported</th></tr>
 
  <tr><td>Chromium</td> <td><a href="//sites.google.com/a/chromium.org/chromedriver/">Chromium</a></td> <td>All versions</td></tr>
- <tr><td>Firefox</td> <td>Selenium</td> <td>4 and newer</td></tr>
+ <tr><td>Firefox</td> <td><a href="https://github.com/mozilla/geckodriver">Mozilla</a></td> <td>54 and newer</td></tr>
```
Comment (manoj9788) [inline start.html:44]: Mozilla instead of Firefox

Comment (AJ-72) [reply start.html:44]: Should the browser name be Mozilla instead of firefox?

Comment (manoj9788) [reply start.html:44]: Mozilla in Maintainer column

Comment (AJ-72) [reply start.html:44]: Sorry, but I don't understand the comment. The change I made was to update the Maintainer column as Mozilla. Its the second TD.

Comment (manoj9788) [reply start.html:44]: Gotcha!

**Comment by manoj9788** [comment]
Thanks! Merged In

**Pull request merged**